app/frontend/wallet/helpers/tokenFormater.ts

```diff
@@ -25,7 +24,7 @@ const aggregateTokensForPolicy = (policyGroup: Array<Token>, policyId: string) =
     )
     .value()
 
-export const aggregateTokens = (tokens: Array<Array<Token>>): Array<Token> =>
+export const aggregateTokens = (tokens: Token[][]): Token[] =>
```

Comment:
`aggregateTokensByPolicy` perhaps would be a better name, especially given that the output type is so opaque

Reply:
ah I see - it actually just takes a list of list of tokens -> I'd create an alias type "TokenBundle" equivalent to "Token[]" to make it clear, and name the function `aggregateTokenBundles` which would imho make it clearer to the reader why nested lists of tokens are being used (actually concealing that technicality from them)

Reply:
Good idea :+1: